### PR TITLE
Feature/todo

### PR DESCRIPTION
**Pull request by oscardelalanza**
To-Do Project
 what's done:

- projects can be created and removed

- each task belongs to a project

- tasks can be created, removed, edited and maked as completed

- the projecs are saved in the browser' local storage

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "no-underscore-dangle": "off",
+      "no-alert": "off",
+      "no-restricted-globals": "off",
+      "import/no-cycle": "off",
+      "no-use-before-define": "off"
+    }
+}
```
Comment (jofer86) [inline .eslintrc.json]: - This file should look like this: 
image
- Please follow the steps described in the [javascript linter repo](https://github.com/microverseinc/linters-config/tree/master/javascript), to configure the linting for your project, make sure you are using the required configuration, and once that is up and running do the necessary linting (if required).

**Review by jofer86 — CHANGES_REQUESTED** [review]
## 👋 👋 Hello Team!!! I'm going to be reviewing this PR, let's take a look, shall we? 👁 👁 👁  

- Well, not much to say here, everything looks fine, functionality is there, code is modulated, although we have one single exception that is preventing me from approving this PR. Please checkout the commented code below, make sure you change what's required and resubmit for another code review- 
--------------------------------------------------------------------
Cheers,
 Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Comment by oscardelalanza** [comment]
list of changes

- linter config changed as requested

- linter checks fixed

- dom module and interface module refactored to avoid dependency cycle

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi Team good job on this one, good job making the changes too :+1: , please see more suggested changes  below :slightly_smiling_face: 

- There should be a defualt project that shows up once the app is run, so any todo created goes in there see

- currently the project is wokring properly there seem to be an error in localstorage.js with getting the length of the todos in a project , please look into that, I think in order to reproduce this you would have to delete the stored content in your localstorage :slightly_smiling_face: 

*Please make changes and request for a re-review*

**RAPHAEL NORIODE** :sunglasses:

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED

Hi Team, good job on making all these changes, please merge and move on, **This is approved** :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**